docs/source/zh/_toctree.yml

```diff
@@ -15,6 +15,8 @@
     title: pipeline教程
   - local: transformers_agents
     title: transformers_agents教程
+  - local: training
```

Comment:
I noticed the `zh` toctree order is off from the `en` one. Would you remind reordering it so they match?

```yml
- sections:
  - local: pipeline_tutorial
    title: pipeline教程
  - local: preprocessing
    title: 预处理
  - local: training
    title: 模型训练教程
  - local: accelerate
    title: 加速分布式训练
  - local: transformers_agents
    title: transformers_agents教程
  title: 教程
```